misc/python/materialize/feature_benchmark/executor.py

```diff
@@ -46,3 +48,83 @@ def Kgen(self, topic: str, args: List[str]) -> Any:
         return self._composition.run(
             "kgen", f"--topic=testdrive-{topic}-{self._seed}", *args
         )
+
+
+class MzCloud(Executor):
+    def __init__(
+        self,
+        composition: Composition,
+        seed: int,
+        mzcloud_url: str,
+        kafka_addr: str,
+        schema_registry_url: str,
+    ) -> None:
+        self._composition = composition
+        self._seed = seed
+        self._mzcloud_url = mzcloud_url
+        self._kafka_addr = kafka_addr
+        self._schema_registry_url = schema_registry_url
+
+    def RestartMz(self) -> None:
+        # We can't restart the cloud, so complain.
+        assert False
+
+    def Reset(self) -> None:
+        print("resetting")
+        self.exec(
+            *[
+                "target/debug/testdrive",
+                f"--materialize-url={self._mzcloud_url}",
+                f"--seed={self._seed}",
+            ],
+            stdin="",
+        )
+
+    def Td(self, input: str) -> Any:
+        return self.exec(
+            *[
+                "target/debug/testdrive",
+                f"--materialize-url={self._mzcloud_url}",
+                f"--kafka-addr={self._kafka_addr}",
+                f"--schema-registry-url={self._schema_registry_url}",
+                "--no-reset",
+                f"--seed={self._seed}",
+                "--backoff-factor=0",
+                "--default-timeout=100s",
```

Comment:
Consider adding `"--initial-backoff=10ms"` here to make cloud and non-cloud testdrive behave identically. The default would otherwise be 50ms